### DISCUSSION TITLE
How to eagerly load images instead of lazy load?

### DISCUSSION THREAD

**Discussion by SamuNatsu** [emilk/egui#4136] [Q&A]
I have some images to be shown, they will be acted like a frame-by-frame animation.

But when I try to show them, it sometimes shows a loading spinner before showing the image, which indicates the image was loaded lazily.

**Comment by sww1235** [comment]
Can you load all the images into byte arrays prior to rendering using `fs::read()`? That way the images are all in memory prior to loading into egui cache and rendering.
